src/pages/access-recovery-confirm/lib/validate-password/validate-password.ts

```diff
@@ -0,0 +1,3 @@
+export function validatePassword(password: string) {
+  return Boolean(password);
```

Comment:
Password `"1"` or `" "` is valid by that function. But I think it isn't safe password.

```suggestion
  return password.trim().length >=8;
```